syntax = "proto3";

package Comentarios;

service ComentariosService {
  rpc PublicarComentario (Comentario) returns (Respuesta);
  rpc ObtenerComentario (ConsultaComentario) returns (returnComentario); 
  rpc ObtenerComentarios (ConsultaComentarios) returns (ListaComentarios);
}

message Comentario {
  int32 evaluacion = 1;
  string comentario = 2;
}

message ConsultaComentario {
  string id = 1;
}

message ConsultaComentarios {
  int32 cantidad = 1;
}

message Respuesta {
  bool exito = 1;
  string mensaje = 2;
}

message returnComentario {
  repeated Comentario comentario = 1;
}

message ListaComentarios {
  repeated Comentario comentarios = 1;
}
